integration-sns-to-lambda/Function.cs

```diff
@@ -0,0 +1,45 @@
+using Amazon.Lambda.Core;
+using Amazon.Lambda.SNSEvents;
+
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace SnsIntegration;
+
+public class Function
+{
+    /// <summary>
+    /// Default constructor. This constructor is used by Lambda to construct the instance. When invoked in a Lambda environment
+    /// the AWS credentials will come from the IAM role associated with the function and the AWS region will be set to the
+    /// region the Lambda function is executed in.
+    /// </summary>
+    public Function()
+    {
+
+    }
+
+
+    /// <summary>
+    /// This method is called for every Lambda invocation. This method takes in an SNS event object and can be used 
+    /// to respond to SNS messages.
+    /// </summary>
+    /// <param name="evnt"></param>
+    /// <param name="context"></param>
+    /// <returns></returns>
+    public async Task Handler(SNSEvent evnt, ILambdaContext context)
+    {
+        foreach (var record in evnt.Records)
+        {
+            await ProcessRecordAsync(record, context);
+        }
+    }
+
+    private async Task ProcessRecordAsync(SNSEvent.SNSRecord record, ILambdaContext context)
+    {
+        context.Logger.LogInformation($"Processed record {record.Sns.Message}");
```

Comment:
The only observation I'd make here is error handling. But I support that would be handled with a DLQ at the Lambda service level. Might be worth putting a comment in to that effect.